lib/kazoo/cluster.rb

```diff
@@ -70,6 +70,26 @@ def topics
       end
     end
 
+    def create_topic(name, partitions: nil, replication_factor: nil)
+      raise ArgumentError, "partitions must be set to a positive integer" if partitions.nil?
```

Comment:
`Integer(partitions) <= 0` :+1:

Normally I'd agree with you on `partitions` being required, but required named arguments are not supported on Ruby 2.0. I'd like to keep this gem compatible with 2.0 because it's the stock version Ruby on OSX.